pcdsdevices/mirror.py

```diff
@@ -1257,10 +1257,10 @@ class FFMirrorZ(FFMirror):
                          kind='normal')
     chin_tail_rtd = Cpt(PytmcSignal, ':RTD:TAIL:TEMP', io='i',
                         kind='normal')
-    # Kill these until MR4 and MR5 K4 implement them.
-    cool_flow1 = None
+
+    cool_flow1 = Cpt(EpicsSignalRO, ':FWM:1_RBV', kind='normal', doc="Axilon Panel Flow Meter Loop 1")
```

Comment:
Where's `cool_flow2`? 😢

Reply:
KBs with 2 mirrors in 1 tank only get one flow meter each.